integration-tests/src/test/java/oracle/kubernetes/operator/ItMonitoringExporter.java

```diff
@@ -677,9 +675,33 @@ private void test13_ReplaceMetricsNameSnakeCaseFalseConfiguration() throws Excep
   }
 
   /**
-   * Try to change monitoring exporter configuration without authentication.
+   * Test to replace monitoring exporter configuration with configuration file with
+   * domainQualifier=true.
    *
-   * @throws Exception exception
+   * @throws Exception if failed to apply configuration or check the expected values.
+   */
+  private void test19_ReplaceMetricsDomainQualifierTrueConfiguration() throws Exception {
+    Assumptions.assumeTrue(FULLTEST);
+    String testMethodName = new Object() {
+    }.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    HtmlPage page =
+        submitConfigureForm(exporterUrl, "replace", configPath + "/rest_domainqualtrue.yml");
+    assertNotNull(page);
+    LoggerHelper.getLocal().log(Level.INFO, "page - " + page.asText());
+    assertTrue(page.asText().contains("domainQualifier"));
+    //1.1.1 does not contain the fix for parsing error in prometheus
+    if (!MONITORING_EXPORTER_VERSION.equals("1.1.1")) {
```

Comment:
what if it is an earlier version? this might give the wrong behavior

Reply:
https://jira.oraclecorp.com/jira/browse/OWLS-80941

Reply:
1.1.1 is earlier version of exporter , I put it because at that time the new release was not posted, to avoid develop breakage. For now the fix is in 1.1.2 so we can remove if condition .